services/productService.js

```diff
@@ -22,7 +24,38 @@ const getProductById = async (req, res, next) => {
   }
 };
 
+const sellProduct = async (req, res, next) => {
+  let product;
+  try {
+    const { id } = req.params;
+    const { body } = req;
+    const { user } = res.locals;
+    if (!body?.productQty) {
+      throw new Error('Missing productQty');
```

Comment:
![Screenshot from 2020-07-28 02-11-02](https://user-images.githubusercontent.com/19937283/88625918-aec43000-d077-11ea-8e56-13297d60deb1.png)

I was unsure where to place this comment but currently when you click 'Sell' the button is disabled for a quantity of '0' but if you increment to '1' then back to '0' it is possible to create a transaction with 0 quantity. Also lets say you close the quantity screen at '3'. When you open it back up its at '3' and go to '0' it is possible to create a transaction with 0 quantity.

I think adding a function to disable the "Confirm" button whenever the quantity state is 0 would proably be best rather than validating quantity on backend.

Reply:
There should always be backend validation, but I think there's a bug here

Reply:
Okay let me know when its fixed and I will approve and merge. Can I only merge this PR and close the other PR? Or should I merge the first PR and then merge this one and keep the changes within this PR.

Reply:
Fixed - was a type issue